charts/timescaledb-multinode/README.md

```diff
@@ -108,6 +122,13 @@ our [TimescaleDB > Tutorial: Scaling out TimescaleDB](https://docs.timescale.com
 to create distributed hypertables and start using multinode TimescaleDB.
 
 ### Connecting from another pod
+If you are connecting from another pod you can disable external access completely by changing the service type from LoadBalancer to ClusterIP. Edit the following in values.yaml:
```

Comment:
as already stated this does not belong here, and the description should also be adapted